### ISSUE TITLE
Make outbound IP ranges more predictable

### ISSUE THREAD

**Issue by jdorothy** [CDCgov/prime-simplereport#2855]
### Description
To work with Experian (and possibly others in the future) we need to provide them a list of potential IP addresses they can expect traffic from so that they can add them to their allowlist. In theory, we can provide that information using the CLI: https://docs.microsoft.com/en-us/azure/app-service/overview-inbound-outbound-ips#find-outbound-ips.

The problem is that these CIDR ranges aren't guaranteed - they'll change if the App Service is recreated or moved to a different resource group, for example. We need a way to guarantee either a) a stable CIDR range or b) a static IP for outbound traffic.

### Proposed solution 
We need to change the way we route outbound traffic. There might be other solutions, but the easiest way seems to be using a NAT Gateway to route outbound traffic through.

### Additional context
We are not affected by CDC networking for outbound traffic so this should be possible entirely within Azure.

**Comment by jdorothy** [comment]
This is deployed and the list of static IP addresses is now known. Closing this in favor of #2882 (actually enabling these changes once Experian is ready).
